backend/qemu.pm

```diff
@@ -244,13 +244,14 @@ sub start_qemu {
         $vars->{BIOS} //= $vars->{UEFI_BIOS};
     }
 
-    if ($vars->{BIOS} && $vars->{BIOS} !~ /^\//) {
-        # Non-absolute paths are assumed relative to /usr/share/qemu
-        $vars->{BIOS} = '/usr/share/qemu/' . $vars->{BIOS};
-    }
-
-    if ($vars->{BIOS} && !-e $vars->{BIOS}) {
-        die "'$vars->{BIOS}' missing, check BIOS\n";
+    foreach my $attribute (qw/BIOS KERNEL INITRD/) {
+        if ($vars->{$attribute} && $vars->{$attribute} !~ /^\//) {
```

Comment:
Ah right. Scratch all that. I set my mind that only relative path is allowed. And I completely ignored that use parse `KERNEL` and `INITRD` in the same loop.